cartridge/failover.old.lua

```diff
@@ -0,0 +1,267 @@
+error([[This module is preserved for reference purpose only,
```

Comment:
Why don't you want to copy it to e.g. wiki page? I don't like approach when you preserve unused code. You have a git and could to move to any point into a history "for reference purpose"